packages/models/user_model/lib/src/user.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:dataclass/dataclass.dart';
+import 'package:json_annotation/json_annotation.dart';
+import 'package:meta/meta.dart';
+
+part 'user.g.dart';
+
+@JsonSerializable()
+@dataClass
+class UserModel extends _$UserModel {
```

Comment:
Any reason for adding `Model` suffix? 
not a blocker, just wondering

Reply:
Because, we don't know how user and prophecy will be represented in data.
I've seperated model that is used between modules. If module will need specific user model, it will be done by composition.
So, local class name User, will not have a conflict with this module.

User
- UserModel
- field
- field
- method

But, yes. It may be not used at all.